CONTRIBUTING.md

```diff
@@ -39,7 +39,6 @@ Separate commits should be used for:
 - API structure (`shared/api: Add XYZ` for changes to `shared/api/`)
 - Go client package (`client: Add XYZ` for changes to `client/`)
 - CLI (`lxc/<command>: Change XYZ` for changes to `lxc/`)
-- Scripts (`scripts: Update bash completion for XYZ` for changes to `scripts/`)
```

Comment:
Does this mean we can also delete the bash completion file in `scripts/bash/lxd-client`?

Reply:
Yes - do we have tests for shell completions that we can run first?

Reply:
I've tested the updated cobra completions locally, after deleting `scripts/bash/lxd-client`, and they work as expected.